ui/src/components/AppHeader.js

```diff
@@ -1,56 +1,69 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
-import AppBar from '@material-ui/core/AppBar';
 import IconButton from '@material-ui/core/IconButton';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
 import MenuIcon from '@material-ui/icons/Menu';
 
 const useStyles = makeStyles(theme => ({
-  appBar: ({ drawerWidth }) => ({
+  appBar: {
     position: 'sticky',
-    ...(drawerWidth && {
-      [theme.breakpoints.up('sm')]: {
-        width: `calc(100% - ${drawerWidth}px)`,
-        marginLeft: drawerWidth,
-      },
-    }),
-  }),
+    [theme.breakpoints.up('sm')]: {
+      display: 'none',
+    },
+    padding: '8px',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
   menuButton: ({ drawerWidth }) => ({
     marginRight: theme.spacing(2),
     ...(drawerWidth && {
       [theme.breakpoints.up('sm')]: {
         display: 'none',
       },
     }),
+    height: '48px',
   }),
-  title: {
-    flexGrow: 1,
+  logo: {
+    height: '60px',
+    width: '200px',
```

Comment:
again, where do these numbers come from? 48px, 60px, 200px

Reply:
These were just arbitrary based on what I thought looked right

Reply:
you also told me that the side nav bar is 240px with a 200px body inside

Reply:
The logo is 200px, but the other content like the nav items isn't necessarily (those are 240px)